dsl/docker-multiarch/images/openjdk.groovy

```diff
@@ -0,0 +1,44 @@
+import vars.multiarch
```

Comment:
I'm really torn on whether to call this file `openjdk.groovy` or `java.groovy`.  This entire repo was misnamed to begin with and I'm really pretty sad that we don't have a clean way to get around the fallout it created.  Regardless, we do need to make sure this image gets built, tagged, and pushed as `PREFIX/java` _not_ `PREFIX/openjdk` (since we need to be as close to faithful to the official images as we can be) -- I'm OK with `openjdk.groovy` if we also add the override in `vars/multiarch.groovy` in `def static image(clsName)` to ensure it tags, builds, and pushes as `java`.